pkg/constants/constants.go

```diff
@@ -29,8 +29,13 @@ const (
 
 	RequestTimeout = 3 * time.Minute
 
-	FujiAPIEndpoint    = "https://api.avax-test.network"
-	MainnetAPIEndpoint = "https://api.avax.network"
+	DeployPublickyLocalMockEnvVar = "DEPLOY_PUBLICLY_LOCAL_MOCK"
+	FujiAPIEndpoint               = "https://api.avax-test.network"
+	MainnetAPIEndpoint            = "https://api.avax.network"
+
+	// this depends on bootstrap snapshot
+	LocalAPIEndpoint = "http://127.0.0.1:9650"
```

Comment:
If I remember correctly, in ANR we have a default of port 9650, but it will take another port if that is already taken (e.g. running fuji or mainnet node). If that is correct, this might be a bit "dangerous" to hardcode like this.

I think the ANR Status should be called and based on that choose the port.

Reply:
ANR is not taking another port on failure. We decided not to do so, and instead keep this ports.
I think we can make a PR to call ANR status if we change that situation,